modules/govuk_mysql/templates/etc/cron.d/xtrabackup_s3

```diff
@@ -0,0 +1,3 @@
+MAILTO=""
+<% @base_backup_cron_minute %> <% @base_backup_cron_hour %> * * * setlock -N /var/run/xtrabackup /usr/local/bin/xtrabackup_s3_base
```

Comment:
Should we include `mysql` in the lock name to make it clear what `xtrabackup` is used for?